packages/manager/modules/iam/src/routes/policies.route.js

```diff
@@ -20,7 +20,19 @@ const state = () => ({
   resolve: {
     ...asResolve(policiesComponent.resolves),
     ...asResolve(resolves),
+    resources: /* @ngInject */ (PolicyService, advancedMode) =>
+      PolicyService.getPolicies({
+        cursor: '',
+        ...(!advancedMode && { readOnly: false }),
+      }).then(({ data }) => data),
   },
+  redirectTo: (transition) =>
+    transition
+      .injector()
+      .getAsync('resources')
+      .then((resources) => {
+        return resources.length > 0 ? false : { state: ROUTES.ONBOARDING };
+      }),
```

Comment:
Same as https://github.com/ovh/manager/pull/9146#discussion_r1169586737 and https://github.com/ovh/manager/pull/9146#discussion_r1169586522